e2e-tests/wiremock/src/main/java/com/foo/wiremock/WireMockApplication.java

```diff
@@ -0,0 +1,15 @@
+package com.foo.wiremock;
+
+
+import static org.hamcrest.Matchers.is;
+import static io.restassured.RestAssured.given;
+
+public class WireMockApplication {
+
+    public String callApi(int port) {
```

Comment:
this is just a function.
should have an actual REST API, in which this fucntion is behind a GET endpoint for example